libtextsecure/sendmessage.js

```diff
@@ -132,6 +133,13 @@ Message.prototype = {
       proto.profileKey = this.profileKey;
     }
 
+    if (this.profile) {
+      const contact = new textsecure.protobuf.DataMessage.Contact();
+      if (this.profile.name)
+        contact.name = this.profile.name;
+      proto.profile = contact;
```

Comment:
Shouldn't we attach a contact/profile only if `this.profile.name` was set?

Reply:
I think we should attach it regardless. this means that in the future when we add a new field that users can set, we can just update a few lines and it will work.